_topic_maps/_topic_map_ms.yml

```diff
@@ -106,15 +106,38 @@ Distros: microshift
 Topics:
 - Name: Using configuration tools
   File: microshift-using-config-tools
+- Name: Cluster authentication
+  File: microshift-kubeconfig
 ---
 Name: Networking
 Dir: microshift_networking
 Distros: microshift
 Topics:
-- Name: Applying networking settings
+- Name: Networking settings
   File: microshift-networking
-- Name: Using a firewall
+- Name: Firewall configuration
   File: microshift-firewall
+# Topics:
```

Comment:
I realize this is commented out, but to be extra careful, I would remove anything extraneous, like lines 120 and 140.